### PR TITLE
JR2-818 Create Job from API

### PR DESCRIPTION
**Pull request by Invizible**

### CODE REVIEW

File: infrastructure-messagebroker-ws-avam-sink/src/main/resources/schema/EgovToAVAM.wsdl [hunk 1]
```diff
@@ -132,7 +132,7 @@
 	 -->
 	<service name="EgovService">
 		<port binding="tns:EgovPortBinding" name="EgovService">
-			<soap:address location="http://localhost:9080/AVAM_Web/services/EgovService"/>
+			<soap:address coordinates="http://localhost:9080/AVAM_Web/services/EgovService"/>
```
Comment (birom) [inline infrastructure-messagebroker-ws-avam-sink/src/main/resources/schema/EgovToAVAM.wsdl]: Thsi should stay location.

File: infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java [hunk 2]
```diff
@@ -69,4 +69,4 @@ public void handleApprovedEvent(JobAdvertisementEvent jobAdvertisementEvent) {
     public void handleRejectedEvent(JobAdvertisementEvent jobAdvertisementEvent) {
         domainEventPublisher.publishEvent(jobAdvertisementEvent);
     }
-}
+}
```
Comment (birom) [inline infrastructure-messagebroker/src/main/java/ch/admin/seco/jobs/services/jobadservice/infrastructure/messagebroker/avam/DefaultAvamService.java]: Newline is missing at the end of the file.

File: infrastructure-messagebroker-ws-avam-sink/src/main/resources/schema/EgovToAVAM.wsdl [hunk 3]
```diff
@@ -132,7 +132,7 @@
...
 		</port>
 	</service>
 </definitions>
```
Comment (birom) [inline infrastructure-messagebroker-ws-avam-sink/src/main/resources/schema/EgovToAVAM.wsdl]: Newline is missing at the end of the file.

File: gradlew [hunk 4]
```diff
@@ -78,14 +78,14 @@ if [ -n "$JAVA_HOME" ] ; then
         die "ERROR: JAVA_HOME is set to an invalid directory: $JAVA_HOME
 
 Please set the JAVA_HOME variable in your environment to match the
-location of your Java installation."
+coordinates of your Java installation."
```
Comment (birom) [inline gradlew]: Do we also want to rename this?

File: infrastructure-messagebroker-ws-avam-sink/src/main/resources/schema/EgovToAVAM.wsdl (hunk 1 above)
Comment (cjpsycos) [reply infrastructure-messagebroker-ws-avam-sink/src/main/resources/schema/EgovToAVAM.wsdl]: IDE did to much work

**Pull request merged**